src/js/controller/controller.js

```diff
@@ -87,6 +87,7 @@ define([
             _setup.on(events.JWPLAYER_READY, _playerReady, this);
             _setup.on(events.JWPLAYER_SETUP_ERROR, this.setupError, this);
 
+            _model.mediaController.on('all', _triggerAfterReady, this);
```

Comment:
@egreaves Should we be concerned about moving this out of the `_playerReady` method and before all the other events for `mediaController` are setup?